api/par-api.yaml

```diff
@@ -135,6 +135,18 @@ paths:
         schema:
           type: integer
           default: 0
+      - name: ordering
+        in: query
+        description: Used for requested ordered responses, this defines the properties
+          to user for ordering the list of Business Rules returned. 
+          Add '-' prefix for descending order.
+        schema:
+          type: string
+          enum:
+            - name
+            - last-modified-date
+            - -name
+            - -last-modified-date
```

Comment:
Are these double-hyphened repeats deliberate? It might just be outside of my experience. Every ordering param has a repeat set, am curious.

Reply:
Ahhhh, the penny drops, it's for reverse ordering...